views/user/new.html

```diff
@@ -0,0 +1,12 @@
+<form action="/user/create" method="post">
+  <label for="user_email">email: </label>
+  <input type="email" name="Email" id="user_email">
```

Comment:
Should we use kebab-case for id naming?

Reply:
@junan I'll deal with this here https://github.com/Lahphim/go-crawler-challenge/pull/38
Because I'm using a package about string manipulate in that PR ✌️